ebml_matroska.xml

```diff
@@ -885,7 +885,9 @@ redundant framing information while preserving versioning and semantics between
 
 Value represents a clockwise rotation, in degrees, around the up vector. This rotation must be applied
 before any `ProjectionPosePitch` or `ProjectionPoseRoll` rotations.
-The value of this field should be in the -180 to 180 degree range.</documentation>
+The value of this field should be in the -180 to 180 degree range.
+
+Setting `ProjectionPoseYaw` to 180 or -180 degrees flips the image horizontally.</documentation>
```

Comment:
Given the vertical flip also involves the same value, we might mention it's a horizontal flip if `ProjectionPoseRoll` is 0 and the Pitch is 0.